app/src/main/java/org/wikipedia/settings/discover/DiscoverSettingsScreen.kt

```diff
@@ -0,0 +1,616 @@
+package org.wikipedia.settings.discover
```

Comment:
Can we change the package name to `recommendedreadinglist`? or, move these classes for settings to the `readinglist.recommended` package?